src/kirby/components/card/card.component.scss

```
@@ -21,6 +21,15 @@ $min-card-width: 304px;
   min-height: $min-card-height;
   min-width: $min-card-width;
   overflow: hidden;
+
+  &[elevation='highlighted'] {
+    box-shadow: #{get-elevation('z4')};
```

No need to wrap the function call (only needed when assigning to css variables):
```suggestion
    box-shadow: get-elevation('z4');
```